release notes/v0.46.0.md

```diff
@@ -0,0 +1,188 @@
+  clients["grpcbin.test.notk6.io:9001"]
+    .connect("grpcbin.test.fakek6.io:9001", params["grpcbin.test.fakek6.io:9001"]);
+}
+...
+```
+</details>
+
+Thanks @chrismoran-mica for the contribution 🙇‍♂️.
+
+### Cloud Output v2 [#3117](https://github.com/grafana/k6/issues/3117)
+
+After years of great service, we decided to refresh the k6 Cloud output introducing a more efficient end-to-end solution for ingesting the generated tests' metrics. The main change regards the protocol used for flushing metrics that is now a binary based payload over HTTP.
+
+The new output reduces the load generators' used resources for tests that produce many metrics. There is no significant difference in the user experience; it's expected to be the same.
+
+The one thing worth highlighting is that the new output is strict about tags and drops tags if they are reserved:
+- `test_run_id` as it is reserved for internal k6 Cloud operations
+- any tag with a key that starts with two underscores (`__`), that is marked by Prometheus convention as reserved
+
+At the moment this is not yet the default Cloud output for the test runs executed from local machines (`k6 run -o cloud`), but it is expected to be transparently enabled in the upcoming weeks.
+
+The full list of related PRs: [#3104](https://github.com/grafana/k6/pull/3104), [#3108](https://github.com/grafana/k6/pull/3108), [#3120](https://github.com/grafana/k6/pull/3120), [#3125](https://github.com/grafana/k6/pull/3125), [#3162](https://github.com/grafana/k6/pull/3162), [#3169](https://github.com/grafana/k6/pull/3169), [#3182](https://github.com/grafana/k6/pull/3182), [#3186](https://github.com/grafana/k6/pull/3186), [#3187](https://github.com/grafana/k6/pull/3187), [#3193](https://github.com/grafana/k6/pull/3193), [#3195](https://github.com/grafana/k6/pull/3195), [#3206](https://github.com/grafana/k6/pull/3206), [#3226](https://github.com/grafana/k6/pull/3226), [#3157](https://github.com/grafana/k6/pull/3157), [#3172](https://github.com/grafana/k6/pull/3172).
+
+### `<big_feature_n>` `#pr`
+
+_what, why, and what this means for the user_
+
+### UX improvements and enhancements
+
+_Format as `<number> <present_verb> <object>. <credit>`_:
+- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
+- [#3176](https://github.com/grafana/k6/pull/3176) Adds a `js/promises` package, which enables extension developers to easily create promises that will be dispatched to the eventloop using the `New` function.
+- [#3181](https://github.com/grafana/k6/pull/3181) Adds a `RunOnEventLoop` method to the `modulestest.Runtime` type, which allows extensions developers to run code on the event loop from their tests.
+- [#xk6-grpc#19](https://github.com/grafana/xk6-grpc/pull/19) Writing to the closed gRPC stream no longer produces the EOF (end of file) error. Additionally, if no error handlers were set up, the error is logged to the console.
```

Comment:
```suggestion
- [xk6-grpc#19](https://github.com/grafana/xk6-grpc/pull/19) Writing to the closed gRPC stream no longer produces the EOF (end of file) error. Additionally, if no error handlers were set up, the error is logged to the console.
```

Not sure how strict we want to be here with consistency, but I guess this could be changed to something like:

Fixes EOF (end of file) error that was produced when writing to a closed gRPC stream. Additionally, if no error handlers are set up, the error is logged to the console.